### PR TITLE
Responsive design

### PR DESCRIPTION
**Pull request by tpaternina**
- Add support for different screen sizes.
- Modify styled components to match mobile them design.
- Migrate individual style pages to a centralized "styled" folder.
- Add component to change the time range of price and volume charts.
- Add currency converter and time range component to Single coin page.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
   StyledLink,
   StyledList,
-  StyledNav,
-} from "./App.styles";
+  NavIcon,
+  NavText,
+  NarrowNav,
+  WideNav,
+  WideNavContainer,
+} from "styled";
```
Comment (yoloOnTheBattlefield) [inline src/App.js:20]: from  styled ?

https://media.giphy.com/media/KGSxFwJJHQPsKzzFba/giphy.gif?cid=ecf05e478lmruftv4wa9a9e7lu44wzm3fcttno1rhmfpe5kq&rid=giphy.gif&ct=g

File: src/components/ChartPriceOverview/ChartPriceOverview.js [hunk 2]
```diff
@@ -1,39 +1,54 @@
...
+        fill: true,
+      },
+    ],
+  };
+
+  const gradient = {
+    id: "responsiveGradient",
+
+    afterLayout: function (chart, options) {
+      const scales = chart.scales;
```
Comment (yoloOnTheBattlefield) [inline src/components/ChartPriceOverview/ChartPriceOverview.js:32]: const {scales}= chart

File: src/styled/Forms.styles.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+  width: 100%;
+
+  @media screen and (min-width: 350px) {
+    margin-bottom: 1rem;
+  }
+
+  @media screen and (min-width: 576px) {
+    margin-bottom: default;
+  }
+`;
```
Comment (yoloOnTheBattlefield) [inline src/styled/Forms.styles.js:78]: can create variable `const tablet = '@media screen and (min-width: 576px)'`

and use it like

const Container = styled.div`
${tablet}{
  styles
}
`

**Review by yoloOnTheBattlefield — COMMENTED** [review]
check the comments

File: src/App.js (hunk 1 above)
Comment (tpaternina) [reply src/App.js:20]: Yeah, that's how I named the folder with all the styled components. Should I change the name to something like "styles" ?

**Pull request merged**